storageClient/storageClient.go

```diff
@@ -74,12 +78,30 @@ func (c *StorageClient) Challenge(ctx context.Context, noteID string) ([]string,
 	return challenges, err
 }
 
+func (c *StorageClient) BulkDeleteByClient(ctx context.Context, clientID uuid.UUID, limit int) (BulkDeleteResponse, error) {
+	path := c.Host + "/internal" + storageServiceBasePath + "/notes/bulk/" + clientID.String()
+	request, err := e3dbClients.CreateRequest("DELETE", path, nil)
+	var resp BulkDeleteResponse
+	if err != nil {
+		return resp, err
+	}
+	urlParams := request.URL.Query()
+	if limit != 0 {
+		urlParams.Set("limit", strconv.Itoa(limit))
+	}
+	request.URL.RawQuery = urlParams.Encode()
+	err = e3dbClients.MakeE3DBServiceCall(c.E3dbAuthClient, ctx, request, &resp)
```

Comment:
We've been slowly chipping away at killing old style auth, why is this internal endpoint using/need to use old style auth?